src/core/BandLimitedWave.cpp

```diff
@@ -26,6 +26,7 @@
 #include "BandLimitedWave.h"
 
 #include <QDataStream>
+#include <QFile>
```

Comment:
Is this unrelated change necessary?

Reply:
I'll try to remove it; I think something changed in the build order after my changes and it did not compile due to the missing `QFile` include. But that was a long time ago and I don't know if it is or should be automatically included from the other header files.

Reply:
So, while it compiled, I don't see any other obvious place from which `QFile` may be included. Maybe it is included by `QDataStream`, but I did not see anything about that in documentation, so it probably should not be assumed to be the case. So I think the new include should stay.

Reply:
I would insist on removing it. The builds compile fine without it, and it can only add to our build times.

Reply:
OK, I'll get rid of it then. I suppose someone else could fix it if / when the compile order changes again and the issue returns.

Reply:
I'm happy for this change to be omitted from this PR since it is indeed out of scope, but I don't think we should remove headers like this in general. This file does use the `QFile` class, so should explicitly include it. It will have a negligible effect on compile times, since modern compilers handle include guards efficiently, and it avoids fragile header dependencies where changes to header files can cause compilation errors in apparently unrelated files.